src/views/answerView.js

```diff
@@ -6,8 +6,13 @@
  */
 export const createAnswerElement = (key, answerText) => {
   const element = document.createElement('li');
-  element.innerHTML = String.raw`
+  const button = document.createElement('button');
+  button.classList.add('answerBtn');
+  element.appendChild(button)
+  button.innerHTML = String.raw`
     ${key}: ${answerText};
   `;
-  return element;
+  button.getAttribute("value", `${key}`);
```

Comment:
I don't think these two lines do anything. It should be `setAttribute` if you want to change it, but I don't think we need to set these attributes in any case.